### ISSUE TITLE
DatePicker: on-change listener not called after deletion of a date (IE)

### ISSUE THREAD

**Issue by DilyanaN** [primefaces/primefaces#6315]
Deletion of a date in a datepicker field is not recognized as a change and the listener is not executed.
This issue occurs only in IE.

**Environment:**
 - PF Version: 8.0.3
 - JSF Version: 2.2
 - Application Server: payara Glassfish 4.1
 - Affected browsers: Internet Explorer

**To Reproduce**
1. Open the page in Internet Explorer.
2. Select or enter a date in the datepicker.
3. Click on the screen outside of the field -> the listener is called.
4. Delete the date.
5. Click on the screen outside of the field -> the listener is NOT called.

**Expected behavior**
The deletion of the date should be recognized as a change and the listener should be called.

**Example XHTML**
```xml
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
    "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:h="http://xmlns.jcp.org/jsf/html"
      xmlns:p="http://primefaces.org/ui"
      xmlns:f="http://xmlns.jcp.org/jsf/core">
<h:head>

</h:head>

<f:view>
  <h:body>
      <h:form>
        
       <div class="ui-g">

          <div class="ui-g-12">
            <p:datePicker yearNavigator="true" showIcon="true" yearRange="1900:2150" pattern="dd.MM.yyyy">
              <p:ajax event="change" listener="#{datePickerMB.onDateChangeListener()}"/>
              <p:ajax event="dateSelect" listener="#{datePickerMB.onDateChangeListener()}"/>
            </p:datePicker>
          </div>

        </div>
      </h:form>

  </h:body>
</f:view>
</html>
```

**Example Bean**
```java
import javax.enterprise.context.RequestScoped;
import javax.inject.Named;

@Named(value = "datePickerMB")
@RequestScoped
public class DatePickerMB {

  public void onDateChangeListener() {
    System.out.println("Listener called!");
  }

}
```

**Comment by melloware** [comment]
@DilyanaN what version of IE you didn't specify?

@tandraschko are we still supporting IE issues?

**Comment by tandraschko** [comment]
maybe @cagataycivici could answer this question. I think we should support IE11 as the there is almost no overhead of supporting it.

**Comment by melloware** [comment]
OK attached is reproducer: 
[pf-6315.zip](https://github.com/primefaces/primefaces/files/5210154/pf-6315.zip)

verified it does not work on PF 8.0 but is fixed in PF 9.0-SNAPSHOT

**Comment by melloware** [comment]
I think it was fixed as part of this ticket: https://github.com/primefaces/primefaces/issues/6034
